abci/example/kvstore/kvstore.go

```diff
@@ -179,50 +181,19 @@ func (app *Application) CheckTx(_ context.Context, req *types.CheckTxRequest) (*
 	if !app.useLanes {
 		return &types.CheckTxResponse{Code: CodeTypeOK, GasWanted: 1}, nil
 	}
-
-	lane := app.assignLane(req.Tx)
-	return &types.CheckTxResponse{Code: CodeTypeOK, GasWanted: 1, Lane: lane}, nil
-}
-
-// assignLane deterministically computes a lane for the given tx.
-func (app *Application) assignLane(tx []byte) uint32 {
```

Comment:
WE can leave it ,it will be added back as soon as we merge the feature branch here.